src/LambdasAPI.ts

```diff
@@ -1,7 +1,34 @@
 import { EthAddress } from 'dcl-crypto'
-import { Profile, RequestOptions } from 'dcl-catalyst-commons'
+import { EntityMetadata, Profile, RequestOptions } from 'dcl-catalyst-commons'
 
 export interface LambdasAPI {
   /** Retrieve / Download */
   fetchProfile(ethAddress: EthAddress, options?: RequestOptions): Promise<Profile>
+
+  fetchProfiles(ethAddresses: EthAddress[], options?: RequestOptions): Promise<Profile[]>
+
+  fetchWearables(filters: WearablesFilters, options?: RequestOptions): Promise<EntityMetadata[]>
+
+  fetchOwnedWearables<B extends boolean>(
+    ethAddress: EthAddress,
+    includeDefinitions: B,
+    options?: RequestOptions
+  ): Promise<OwnedWearables<B>>
+}
+
+export type WearablesFilters = {
+  collectionIds?: string[]
+  wearableIds?: string[]
+  textSearch?: string
+}
+
+export type OwnedWearables<B extends boolean> = (B extends false
```

Comment:
This is why I love typescript. Depending on whether the `includeDefinitions` param is true or false, the return type changes